package.json

```diff
@@ -3,15 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^7.14.2",
+    "firebase": "^7.14.3",
+    "firestorter": "^2.0.1",
     "husky": "^3.1.0",
     "lint-staged": "^9.5.0",
+    "mobx": "^5.15.4",
+    "mobx-react": "^6.2.2",
```

Comment:
@nabilkaz @Tanzi11 I don't see `firestorter` and `mobx` being used anywhere in here, did you forgot to remove it? or did I just miss anything here  😅?

Reply:
Nothing that I need. To be honest I don’t know what they are 😅. @Tanzi11 did you need these or can we delete them?

Reply:
Removed without issues 👍🏽